modules/preview-react/table/lib/TableBody.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import {Grid, GridProps} from '@workday/canvas-kit-react/layout';
+import {createComponent, styled, StyledType} from '@workday/canvas-kit-react/common';
+
+const StyledBody = styled(Grid.as('tbody'))<StyledType>({
+  'tr:last-of-type td': {
+    borderBottom: 'none',
+  },
+  'tr:last-of-type th': {
+    borderBottom: 'none',
+  },
+  'td:last-of-type': {
+    borderRight: 'none',
```

Comment:
do you need this borderRight? I'm trying to test it and I don't see it impacting anything?

Reply:
If you're adding just border bottom and the table has border all around, I don't think you need to get border right to 0?

Reply:
So what this is doing is, it's removing the `border-right` for the last cells of a row. 

So if a user was to apply a `border-right` so have vertical dividers between cells, then this would remove the border from the right of the last cell so there isn't a double border.

Reply:
This will also be updated to include left and right borders